src/Application/Controller.cpp

```diff
@@ -355,18 +365,18 @@ MakeJsonDocFn Controller::createMakeJsonDocFn(const char* pJsonSchemaFilePath)
     auto jsonDocPtr = createJsonDoc();
     jsonDocPtr->setContent(designJson);
 
-    if (!jsonSchemaFilePath.empty())
-    {
-      SchemaValidJsonDocument::ValidatorPtr validator;
-      std::ifstream schemaIfs(jsonSchemaFilePath);
-      json schema = json::parse(schemaIfs);
-      validator.reset(new JsonSchemaValidator);
-      validator->setRootSchema(schema);
+    // if (!jsonSchemaFilePath.empty())
```

Comment:
So are we disabling json schema validation forever? Or this is just a temporary workaround?

Reply:
> So are we disabling json schema validation forever? Or this is just a temporary workaround?

SchemaValidJsonDocument be improved in a future PR.